src/shared/components/palette/utils/date.pipe.spec.ts

```diff
@@ -54,8 +54,6 @@ describe('DatePipe', () => {
   it('should render correct date if UTC date in yyyy-mm-ddThh:mm:ss PM format', () => {
     const firstPass: string = datePipe.transform('2017-07-26T20:10:05', 'local', null);
     expect(firstPass).toBe('26 Jul 2017, ' + getExpectedHour(8) + ':10:05 PM');
-    const secondPass: string = datePipe.transform(firstPass, 'local', null);
```

Comment:
I added this test in to make sure the pipe doesn't break if it gets a date a second time. This was because there were circumstances on the Case details tabs where dates were being parsed twice and weren't rendering correctly.

If this has been removed because it's failing, that will regress the fix.